Lib/test/test_unittest/testmock/testthreadingmock.py

```diff
@@ -145,14 +176,14 @@ def test_wait_until_any_call_positional(self):
             self.run_async(something.method_1, 3, delay=0.3)
             self.assertNotIn(call(1), something.method_1.mock_calls)
 
-            something.method_1.block_until_any_call(1)
+            something.method_1.wait_until_any_call(1)
```

Comment:
The smaller API surface of losing the separate `block_` method is certainly appealing!
Doesn't this fail now that the parameter is keyword-only?